calibration/CTDMOG/CGINS-CTDMOG-13590__20150612.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13590,CC_ptempa0,150.1421,
 37-13590,CC_ptempa1,-0.06277276,
 37-13590,CC_ptempa2,-7.56818e-07,
-37-13590,CC_wbotc,4.0017e-07,
+37-13590,CC_wbotc,4.001652e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT